### ISSUE TITLE
Node version in Readme

### ISSUE THREAD

**Issue by rohansaw** [openforis/collect-earth-online#1766]
The Readme states that using "Node 10" is a requisite for building the project. However the `vite` version specified in package.json requires node " "node": "^14.18.0 || >=16.0.0"".

It would be great to update this, since I am trying to find a version in which all the dependencies are solvable.

**Comment by a-luz** [comment]
This is true Rohan, the readme does need some updates, I'll be updating it this week!
